plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/DockerConnector.java

```diff
@@ -168,6 +173,55 @@ public Version getVersion() throws IOException {
     }
 
     /**
+     * Method returns list of docker containers, include non-running ones.
+     *
+     * @throws IOException
+     *         when problems occurs with docker api calls
+     */
+    public List<ContainerListEntry> listContainers() throws IOException {
+        return listContainers(ListContainersParams.from().withAll(true));
+    }
+
+    /**
+     * Method returns list of docker containers which was filtered by query parameters from {@link ListContainersParams}
```

Comment:
ListContainersParams says nothing about query params, so don't use query keyword in explanation